yt/frontends/gamer/cfields.pyx

```diff
@@ -0,0 +1,215 @@
+# distutils: include_dirs = LIB_DIR
+# distutils: libraries = STD_LIBS
+cimport cython
+cimport libc.math as math
+cimport numpy as np
+import numpy as np
+
+np.import_array()
```

Comment:
I don't think we need this in our library code

Reply:
gone